webrtc.html

```diff
@@ -1816,9 +1795,20 @@
                         </ol>
                       </li>
                       <li>
+                        <p>If <code><var>candidate</var>.ufrag</code> is not
+                        equal to any ufrag present in the corresponding
+                        <a>media description</a> of an applied remote
+                        description, reject <var>p</var> with an
```

Comment:
I think that saying that the ufrag MUST be present in at least one of the current and pending remote descriptions is fine.